rust/src/http2/detect.rs

```diff
@@ -604,68 +684,193 @@ pub unsafe extern "C" fn rs_http2_tx_get_header_value(
     return 0;
 }
 
-fn http2_escape_header(hd: &parser::HTTP2FrameHeaders, i: u32) -> Vec<u8> {
+fn http2_escape_header(blocks: &Vec<parser::HTTP2FrameHeaderBlock>, i: u32) -> Vec<u8> {
```

Comment:
This should probably be `blocks: &[parser::HTTP2FrameHeaderBlock]`.

Reply:
both works, changing for your version :-)